### ISSUE TITLE
No Sound

### ISSUE THREAD

**Issue by JohnKendrick** [Taiko2k/TauonMusicBox#799]
Hi I have been using TauonMusicBox for a while with no problems.  But now I get no sound.  The equaliser shows there is sound, but nothing coming through the speakers.  If I use other players I have no problem.

My system is arch linux, software installed through the AUR.  The version number is 7.3.2.  I tried re-installing and recompiling all the python modules I could find with pip.  Volume is turned up full.  I am playing through a Scarlett 2i4 sound card.  The pulse audio mixer control shows that play back from Tauon Music Box: miniaudi1 is on.  There is a very low signal showing underneath the volume slider.

I'm not sure what else to do.  I installed tauon on another machine and everything is working as expected.

Any suggestions?
Thanks John

**Comment by JohnKendrick** [comment]
I have default selected under Audio, I have also tried the other entries
Scarlett 2i4 surround 4.0 and
Builtin Audio Digital Stereo (HDMI2)
![tauon-1](https://user-images.githubusercontent.com/1144794/181802194-fc282907-623e-4acd-b7fb-560bcd70f5af.png)

**Comment by Taiko2k** [comment]
Hmm, I have received other reports of Tauon not having sound as well. Seems to happen with more esoteric sound cards maybe. Might be hard to get to the bottom of it.

If you start tauon from terminal (just type `tauon`), then start to play a track, it should print what samplerate it chose to output. What rate does it say for you?

**Comment by JohnKendrick** [comment]
Output says;
ph: Using samplerate 44100hz

**Comment by JohnKendrick** [comment]
I've found what I think is the problem.  Although I haven't come up with a solution.  I noticed that there was a report of a problem with miniaudio playback.  The details are here;
https://github.com/mackron/miniaudio/issues/480
I used the test example quoted there and I see exactly the same behaviour.  If I run with 2->4 channels which is the pulse audio setting for the device, the having STEREO defined in the test gives no sound.  If I remove STEREO I get a signal through.  So the problem is with the miniaudio library.
